apps/demos/Demos/Chat/Overview/React/App.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+
+import { Chat, ChatTypes } from 'devextreme-react/chat';
+import { MessageEnteredEvent } from 'devextreme/ui/chat';
+
+import service from './data.ts';
+
+export default function App() {
+    const [ userChatTypingUsers, setUserChatTypingUsers ] = useState([]);
+    const [ supportChatTypingUsers, setSupportChatTypingUsers ] = useState([]);
```

Comment:
Why did we add <ChatTypes.Message[]> for useState on the next line but miss it for typingUsers? Should we add typing here?

Reply:
b8d89713faef78d287add7244259689e381718d2